lib/matplotlib/dates.py

```diff
@@ -1426,7 +1441,9 @@ def autoscale(self):
         ymin = self.base.le(dmin.year)
         ymax = self.base.ge(dmax.year)
         vmin = dmin.replace(year=ymin, **self.replaced)
+        vmin = vmin.astimezone(self.tz)
```

Comment:
Is this acutally different from he above cases?

Reply:
Ummm, hmmm, not sure this was properly tested....